docs/deployment/pull-backup.rst

```diff
@@ -1,9 +1,22 @@
 .. include:: ../global.rst.inc
 .. highlight:: none
 
+=======================
 Backing up in pull mode
 =======================
 
+Normally the borg client pushes the backup to a backup server using SSH as a
+transport. This means, the machine holding the data to be backed up, initiates
+the connection to the backup server. If you want to backup a remote server (in
+the borg perspective this would be the client) to your local hard drive in your
+PC (in the borg perspective this would be the backup server), the remote server
+needs a way to connect to your PC. Normally this is not possible.
+
+There are two ways to work around this issue:
```

Comment:
This is overcomplicating the matter with specific detail that is neither general nor relevant - being local or remote, being a PC or something else and being connected via a router (with about any system in the internet is :wink:) has nothing to do with the question of whether system A can connect to system B, but not the other way around.
Which only is a possible technical motivation for pull mode, it might as well be a policy reason to strive for backup server triggered backup runs.
So maybe:
"Typically the borg client connects a backup server using SSH as a transport when initiating a backup. This is referred to as push mode.
If you however require the backup server to initiate the connection or prefer it to initiate the backup run, one of the following workarounds is required to allow such a pull mode setup."

Reply:
Sounds good!

Reply:
I replaced the introduction, and added a sentence.